setup/bootstrap.sh

```diff
@@ -17,34 +17,34 @@ if [[ $EUID -ne 0 ]]; then
 fi
 
 # Clone the Mail-in-a-Box repository if it doesn't exist.
-if [ ! -d $HOME/mailinabox ]; then
+if [ ! -d "$HOME/mailinabox" ]; then
 	if [ ! -f /usr/bin/git ]; then
-		echo Installing git . . .
-		apt-get -q -q update
-		DEBIAN_FRONTEND=noninteractive apt-get -q -q install -y git < /dev/null
+		echo "Installing git . . ."
+		apt-get -qq update
```

Comment:
No, because the `setup/functions.sh` file with the `hide_output` function, has not yet been downloaded. This file/script is installing `git` so it can clone the repository.